Packs/F5/Integrations/F5/integration-F5.yml

```diff
@@ -130,19 +158,61 @@ script:
             }
             return 'Gevald!!';
         default:
-            return sendRequest(getMethod(command), getUrl(command, args), buildBody(command, args));
+            body = sendRequest(getMethod(command), getUrl(command, args), buildBody(command, args));
+            md = tableToMarkdown("F5 Results", body);
+            cmd_arr = command.split('-');
+            key = "F5." + cmd_arr[cmd_arr.length - 1];
+            entry = {
+                'Type' : entryTypes.note,
+                'Contents': body,
+                'ContentsFormat' : formats.json,
+                'HumanReadable': md,
+                'ReadableContentsFormat' : formats.markdown
+            };
+            entry['EntryContext'] = {}
+            entry['EntryContext'][key] = body
+
+            return entry;
     }
   type: javascript
   commands:
   - name: f5-create-policy
-    arguments: []
+    arguments:
+    - name: name
+      required: true
+      description: The name of the policy to create
+    - name: description
+      required: true
+      description: The description of the policy
```

Comment:
Does it have to be required,
Because until now it did not exist at all so it did not work?
And that might make backward compatibility

Reply:
Nice catch, doesn't look like it is required.  Will remove that config

Reply:
name on "f5-create-policy" and name, action, placeBefore on "f5-create-rule"  have to be required?
Because might brake backward compatibility